src/scripts/install_machine.py

```diff
@@ -162,8 +161,7 @@ def prepare_installation_source(args):
         params += ['--install', f"os={args['os']}"]
     elif args['sourceType'] in ['disk_image', 'cloud']:
         params.append("--import")
-    elif ((args['source'][0] == '/' and os.path.isfile(args['source'])) or
-            (args['sourceType'] == 'url' and args['source'].endswith(".iso"))):
+    elif args['source'].endswith(".iso") and args['os'].startswith('win'):
```

Comment:
Hardcoding image names that way is really ugly -- could that have a `# HACK` comment pointing to https://github.com/virt-manager/virt-manager/issues/296 or a similar bz?

Reply:
I am checking here the prefix of the operating system short name, this is a list that osinfo-db has and is not random. So the only Oses that start with win are versions windows.
 
virt-manager does not support unattended installs and it always uses --cdrom for iso files exactly I we had it before.

I needed to make the change, because as Fabiano pointed out in the BZ here https://bugzilla.redhat.com/show_bug.cgi?id=1868594 in order for unattended installs to work we need to use --location for linux, however for WIndows we need to keeo using --cdrom. That's really confusing and badly documented that's why I opened -https://github.com/virt-manager/virt-manager/issues/296

If you think that's too vaguely explained I can wait till they answer in the issue here https://github.com/virt-manager/virt-manager/issues/296.

Reply:
I can add an inline comment with this issue of course, I just wonder if that's enough even.

Reply:
I'm mostly worried that in a few months we'll look at this and can't make sense of "why are we using --cdrom and --location like that"? Some pointer to the details would be useful there. Maybe point to that bugzilla and virt-manager#296?